internal/ingress/controller/template/configmap.go

```diff
@@ -150,6 +152,25 @@ func ReadConfig(src map[string]string) config.Configuration {
 		}
 	}
 
+	// Verify that the configured global auth is parsable as URL. if not, set the default value
+	if val, ok := conf[globalAuthURL]; ok {
+		delete(conf, globalAuthURL)
+
+		authURL, err := url.Parse(val)
+		if err != nil {
+			glog.Warningf("Global auth location denied, reason: url is not")
+		}
+		if authURL.Scheme == "" {
+			glog.Warningf("Global auth location denied, reason: url scheme is empty")
+		} else if authURL.Host == "" {
+			glog.Warningf("Global auth location denied, reason: url host is empty")
+		} else if strings.Contains(authURL.Host, "..") {
```

Comment:
Is this some kind of edge case where `url.Parse` does not err?

Reply:
This was taken from auth-url annotation parser. 
https://github.com/kubernetes/ingress-nginx/blob/master/internal/ingress/annotations/authreq/main.go#L137
The idea behind was to follow the syntax and style of the existing solution

Reply:
Yes, it is the case `url.Parse` does not catch.